java.asciidoc

```diff
@@ -0,0 +1,298 @@
+= Pony for Java
```

Comment:
This reference is a little more complete: http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/
It seems `=` is the level 0 document title, and then the headers start at level 1 `==`.

Looking at this example: https://github.com/dakeshi/gitbook-asciidoc-startkit
It seems they start each chapter with a level 0 `=` title.

Also, it seems a little more standard to make the top-level README and SUMMARY as .adoc files. But it seems you can mix markdown and asciidoc fine, so that's up to you.